src/Pim/Bundle/CatalogBundle/tests/integration/ElasticSearch/PimCatalogTextareaIndexConfigurationIntegration.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+namespace Pim\Bundle\CatalogBundle\tests\integration\Elasticsearch;
+
+/**
+ * This integration tests checks that given an index configuration and some products indexed
+ * the text area research are consistent.
+ *
+ * @author    Samir Boulil <samir.boulil@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class PimCatalogTextareaIndexConfigurationIntegration extends AbstractIndexConfigurationIntegration
+{
+    public function testStartWithOperator()
+    {
+        $query = $this->createSearchQuery([
+            'query' => [
+                'bool' => [
+                    'filter' => [
+                        'query_string' => [
+                            'default_field' => 'description-text',
+                            'query'         => 'an*',
+                        ],
+                    ],
+                ],
+            ],
+        ]);
+
+        $productsFound = $this->getSearchQueryResults($query);
+
+        $this->assertProducts($productsFound, ['product_2', 'product_5']);
+    }
+
+    public function testContainsOperator()
+    {
+        $query = $this->createSearchQuery([
+            'query' => [
+                'bool' => [
+                    'filter' => [
+                        'query_string' => [
+                            'default_field' => 'description-text',
+                            'query'         => '*My*',
+                        ],
+                    ],
+                ],
+            ],
+        ]);
+
+        $productsFound = $this->getSearchQueryResults($query);
+
+        $this->assertProducts($productsFound, ['product_1']);
+    }
+
+    public function testDoesNotContainOperator()
```

Comment:
In ES bundle, this operator was implemented with a `must_not: match_phrase` operator. I switched for a query string to match the TextFilter::DOES_NOT_CONTAIN query. 

In the end, this switch allows to use one PQB filter for both Text and TextArea.